docs/1-qms/0-quality-manual-policy-objectives.md

```diff
@@ -0,0 +1,191 @@
+
+## 1. Scope
+
+The QMS described in this Quality Manual applies to all products of
+{{device_manufacturer}}.
+
+### Role of Company
+
+{{device_manufacturer}} is a manufacturer of Medical Devices.
+
+### Applicable Standards
+
+| Standard                                             | Why Applicable?                                                    |
+| ---------------------------------------------------- | ------------------------------------------------------------------ |
+| Therapeutic Goods Act 1989                           | The Act for all Medical Devices Manufacturers in Australia         |
+| Therapeutic Goods (Medical Devices) Regulations 2002 | The regulations for all Medical Devices Manufacturers in Australia |
+| MDR (2017/745/                                       | Regulation for all Medical Device Manufacturers in the EU          |
+| ISO 13485:2016                                       | QMS utilised to meet the TGA essential principles                  |
+| ISO 14971:2019                                       | Risk management for medical devices                                |
+| IEC 62304:2006                                       | Software development for medical devices                           |
+| IEC 62366-1:2015                                     | Usability evaluation for medical devices                           |
+
+### Exclusions
+
+The following sections of ISO 13485:2016 will be excluded due to the product
+being stand-alone software:
+
+- 6.4.2 Contamination control
+- 7.5.2 Cleanliness of product
+- 7.5.5 Particular requirements for sterile medical devices
+- 7.5.7 Particular requirements for validation of processes for sterilization
+  and sterile barrier systems
+- 7.5.9.2 Particular requirements for implantable medical devices
+
+## 2. Quality Policy & Objectives
+
+### Quality Policy
+
+<!-- > Describe what your company is about, specifically, its mission and things
+> which are important for it. Maybe you're developing software for patients
+> with a certain disease and your goal is to improve their lives. -->
+
+{{device_manufacturer}} aims to make AI assisted radiotherapy cancer treatments
+accessible to all. With the goals to reduce treatment errors while improving
+overall treatment efficacy and efficiency.
+
+### Quality Objectives
+
+<!-- > Whatever policy you outlined above, now you need to make it measurable by
+> defining objectives which can be tracked. Those objectives should not (only)
+> refer to the quality of your devices but the quality of your QMS and the
+> overall work of your organization. > Typical examples are: hiring excellence
+> in staff, providing best-of-class device performance, high standards of
+> customer satisfaction, etc. In a next step (see short-term goals), those are
+> narrowed down to concrete measures like for example the monthly number of
+> user complaints. -->
+
+When tracking the quality of the AI autocontouring software its
```

Comment:
This only addresses the quality of the "algorithm".
It does not address the quality of the software (defects, uptime, customer satisfaction related to non-functional requirements, such as performance, ease of use, ease of installation, etc.)

Reply:
Yup, I'll pull this out into an issue to address when I am looking at this document again.